app/javascript/packages/document-capture/components/in-person-location-step.spec.tsx

```diff
@@ -18,6 +20,20 @@ describe('InPersonLocationStep', () => {
       .withArgs(LOCATIONS_URL)
       .resolves({
         json: () => Promise.resolve([{ name: 'Baltimore' }]),
+      } as Response)
```

Comment:
Just want to note that this now pierces the abstraction of `request` to stub out the underlying `fetch` calls in a way we should probably want to avoid, i.e. because `InPersonLocationStep` now only knows that it can call `request` with a particular set of arguments and expect to receive a return value of some type, I would expect the stubbing to reflect this.

It can be kinda tricky to stub out imported modules, but we've done something similar with `@18f/identity-analytics` that we could use as prior art?

https://github.com/18F/identity-idp/blob/b4dbd477d881de4122fff59a3ffa972fca00a1f5/app/javascript/packages/password-toggle/password-toggle-element.spec.ts#L4

https://github.com/18F/identity-idp/blob/b4dbd477d881de4122fff59a3ffa972fca00a1f5/app/javascript/packages/password-toggle/password-toggle-element.spec.ts#L53

Reply:
Another approach could be to stub out the endpoints we expect to be called and completely bypass _how_ they'd be called, using a package like [`nock`](http://npmjs.com/package/nock).